app/knowledge_panels.py

```diff
@@ -225,12 +224,16 @@ async def wikidata_kp(self):
 
             html.append(
                 {
-                    "element_type": "links",
-                    "link_element": {
-                        "wikipedia": val.wikipedia_relation,
-                        "image_url": val.image_url,
-                        "open_street_map": val.OSM_relation,
-                        "INAO": val.INAO_relation,
+                    "element_type": "text",
+                    "text_element": {
+                        "html": (
+                            "<ul>"
+                            f"<li><a href={val.wikipedia_relation}'>wikipedia</a></li>"
+                            f"<li><a href={val.image_url}>wikidata thumbnail</a></li>"
+                            f"<li><a href={val.OSM_relation}>OpenSteetMap Relation</a></li>"
+                            f"<li><a href={val.INAO_relation}>INAO relation</a></li>"
```

Comment:
There are different problem:
1. the first link lacks opening quote
2. image_url is a image url so we should display it as an image instead :-) (using <img src="...">)
3. you forgot to add quotes around urls 
4. for OpenStreet map relation and INAO you should condition it by the fact that they should not be none or ""
eg. for http://localhost/knowledge_panel?lang_code=fr&facet_tag=category&value_tag=en:beers

I think it's worth building a list of string and using "".join here.

Reply:
indeed!